cmake/f3dPlugin.cmake

```diff
@@ -263,4 +283,24 @@ macro(f3d_plugin_build)
       EXCLUDE_FROM_ALL)
   endforeach()
 
+  # JSON
+  string(JSON F3D_PLUGIN_JSON
+    SET "${F3D_PLUGIN_JSON}" "name" "\"${F3D_PLUGIN_NAME}\"")
+
+  string(JSON F3D_PLUGIN_JSON
+    SET "${F3D_PLUGIN_JSON}" "description" "\"${F3D_PLUGIN_DESCRIPTION}\"")
+
+  string(JSON F3D_PLUGIN_JSON
+    SET "${F3D_PLUGIN_JSON}" "version" "\"${F3D_PLUGIN_VERSION}\"")
+
+  string(JSON F3D_PLUGIN_JSON
+    SET "${F3D_PLUGIN_JSON}" "type" "\"${F3D_PLUGIN_TYPE}\"")
+
+  set(F3D_PLUGIN_JSON_FILE "${CMAKE_CURRENT_BINARY_DIR}/${F3D_PLUGIN_NAME}.json")
+
+  file(WRITE "${F3D_PLUGIN_JSON_FILE}" "${F3D_PLUGIN_JSON}")
+
+  install(FILES "${F3D_PLUGIN_JSON_FILE}"
+      DESTINATION "share/f3d/plugins"
+      COMPONENT plugin)
```

Comment:
I think this is part of sdk, not plugin ?

Reply:
I'll rework the installation components later, it's messy right now. Out of scope of this PR though.

Reply:
fair enoug